### PR TITLE
Add basic plugin support.

### PR DESCRIPTION
**Pull request by jimmiebtlr**
Move models to packages.
Add plugin package, and insert plugin points in UI.

Would love some discussion on how to improve.
Issue #331 

Adds some simple entry points for a plugin.  Not terribly clean, but setting a template name will result in that template being rendered in the section.

    Wekan.registerPlugin({
      cardAction: "CardTimeToggle",
      listHeader: "ListTimeHeader",
      boardOverview: "TimeBoardOverview"
    });

See an example of a plugin at https://github.com/jimmiebtlr/wekan-time.

[Review on Reviewable](https://reviewable.io/reviews/wekan/wekan/332)

### CODE REVIEW

**Comment by jimmiebtlr** [comment]
It runs, but looks like I have some cleanup for eslint to do.  What's up with the license?

File: packages/activities/package.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Package.describe({
+  name: 'wekan:activities',
+  version: '0.0.1',
```
Comment (mquandalle) [inline packages/activities/package.js]: We should keep all the package version number in sync with the main app, ie `0.9.0`.

File: packages/activities/package.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  // Brief, one-line summary of the package.
+  summary: '',
+  // URL to the Git repository containing the source code for this package.
+  git: '',
+  // By default, Meteor will default to using README.md for documentation.
+  // To avoid submitting documentation, set this field to null.
+  documentation: 'README.md'
```
Comment (mquandalle) [inline packages/activities/package.js]: Let's remove lines 4 to 10.

**Comment by mquandalle** [comment]
Hi @jimmiebtlr,

Thank you very much for this PR. My intuition is that we should be able to customize the set of plugins we use at the board level.

I will probably merge this in a separate branch so I can continue experiment with this architecture. The CLA is pretty common is open-source project, it says that you keep the copyright of your work but you give me a license to give me the right use it.

**Comment by jimmiebtlr** [comment]
I can definitely see the use case for plugins by board.  The specific use case I intended would actually be site wide, but possibly dependent on the user role.

**Comment by mquandalle** [comment]
We are not going to use Meteor packages as the basis of our plugin system, but instead will rely on NPM. I published an overview of the desired plugin system here: https://discuss.wekan.io/t/rfc-the-plugin-system/132/1

As you worked on such a system for Wekan, I would be curious to read your thoughts on my proposal.

**Pull request closed without merging**